### ISSUE TITLE
[Feature] - Support paging for repositories in container registry

### ISSUE THREAD

**Issue by F21** [vultr/vultr-cli#454]
**Is your feature request related to a problem? Please describe.**
When listing the repositories in a container registry, the results returned are pagable, however, the cli doesn't seem to support paging or cursors:
```
❯ docker run -e VULTR_API_KEY vultr/vultr-cli:latest container-registry repository list REDACTED --per-page=1
Error: unknown flag: --per-page
❯ docker run -e VULTR_API_KEY vultr/vultr-cli:latest container-registry repository list REDACTED  --cursor=xxx
Error: unknown flag: --cursor
```

**Describe the solution you'd like**
Implement paging and cursors for container registry

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by optik-aper** [comment]
That would be a limitation of the [API](https://www.vultr.com/api/#tag/Container-Registry/operation/list-registries), I'm afraid.

**Comment by F21** [comment]
I think the API supports paging as it contains the `meta` object in the response.

**Comment by optik-aper** [comment]
Yeah, in the response but it doesn't appear the controller has it on the request.  I'll look into it on the API side.
